### ISSUE TITLE
Rule `no-direct-event-calls-in-components`

### ISSUE THREAD

**Issue by sergeysova** [effector/eslint-plugin#111]
It can be useful for SSR or SPA with testing approaches.

Should ban direct calls of effector events inside React/Solid components.

Fail:

```ts
// no-direct-event-calls-in-components: 'error'
import { somethingHappened } from './model'; // Event

function Example() {
  // Should suggest to wrap it with `useEvent` from "effector-react/scope"
  return <div onClick={() => somethingHappened()} />
}
```

Okay:

```ts
// no-direct-event-calls-in-components: 'error'
import { useEvent } from 'effector-react/scope';
import { somethingHappened } from './model'; // Event

function Example() {
  const handleClick = useUnit(somethingHappened);
  // Should suggest to wrap it with `useEvent` from "effector-react/scope"
  return <div onClick={() => handleClick()} />
}
```

**Comment by kireevmp** [comment]
Implemented for React as `mandatory-scope-binding` ([doc](https://eslint.effector.dev/rules/mandatory-scope-binding))
